lib/src/supabase.dart

```diff
@@ -52,7 +53,7 @@ class Supabase {
       _instance.log('***** Supabase init completed $_instance');
 
       await SupabaseAuth.initialize(
-        localStorage: localStorage ?? const LocalStorage(),
+        localStorage: localStorage ?? const HiveLocalStorage(),
```

Comment:
I haven't got much dart 2.0 experience, but could the default `?? const HiveLocalStorage()` be moved up to the static initialize method parameter? 

My thought process here is, am I able to disable persistence or will it be forced with this package?

Reply:
you can create a custom LocalStorage implementation that does nothing. That way, it'll never persist the session. It's a great idea to have it implemented by default on the package

Reply:
I added the `EmptyLocalStorage` class to disable session persistance